packages/authorization-process/src/services/validators.ts

```diff
@@ -73,6 +74,20 @@ export const assertOrganizationIsPurposeConsumer = (
   }
 };
 
+export const assertOrganizationIsDelegate = (
+  organizationId: TenantId,
```

Comment:
Fors consistency with agrreement & purpose services, and also to avoid misuse making the signature more specific
```suggestion
  authData: AuthData,
```